bin/reth/src/args/utils.rs

```diff
@@ -15,6 +15,13 @@ use std::{
 #[cfg(feature = "optimism")]
 use reth_primitives::{BASE_GOERLI, BASE_MAINNET};
 
+#[cfg(feature = "optimism")]
+/// Chains supported by op-reth. First value should be used as the default.
+pub const ALLOWED_CHAINS: &[&str] = &["base", "base_goerli"];
+#[cfg(not(feature = "optimism"))]
+/// Chains supported by reth. First value should be used as the default.
+pub const ALLOWED_CHAINS: &[&str] = &["mainnet", "sepolia", "goerli", "holesky", "dev"];
```

Comment:
I'd like to rename this to `SUPPORTED_CHAINS`.

Reply:
done